### PR TITLE
feat: LT-1074 copy existing parties

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: applications/urls.py [hunk 1]
```diff
@@ -96,4 +97,6 @@
         documents.GeneratedDocument.as_view(),
         name="application_generated_document",
     ),
+    # Copy parties
```
Comment (efahy) [inline applications/urls.py]: This comment is out of date?  The endpoint just returns existing parties..

File: applications/views/existing_parties.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+from conf.authentication import ExporterAuthentication
+from parties.models import Party
+from parties.serializers import PartySerializer
+
+
+class ExistingParties(generics.ListCreateAPIView):
+    authentication_classes = (ExporterAuthentication,)
+    serializer_class = PartySerializer
+
+    def get_queryset(self):
```
Comment (efahy) [inline applications/views/existing_parties.py]: Perhaps a wee docstring comment here or for the View class summarising what it does, with particular reference to the filtering?

File: applications/tests/tests_existing_parties.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
+from django.urls import reverse
+from parameterized import parameterized
+from rest_framework import status
+
+from parties.models import Party, EndUser
+from static.countries.models import Country
+from test_helpers.clients import DataTestClient
+
+
+class GetExistingPartyTests(DataTestClient):
```
Comment (efahy) [inline applications/tests/tests_existing_parties.py]: "Parties" technically

File: applications/tests/tests_existing_parties.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+    def test_get_existing_parties(self):
+        response = self.client.get(self.url, **self.exporter_headers)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.data["results"]), Party.objects.count())
+
+    @parameterized.expand(
+        [("name", "Abc", 2), ("address", "123 abc st.", 1), ("address", "456 abc st.", 1),]
+    )
```
Comment (efahy) [inline applications/tests/tests_existing_parties.py]: I would add another set of parameters that filters everything out.  Should still give back a 200 but with zero results, right?

File: applications/tests/tests_existing_parties.py (hunk 3 above)
Comment (BenAndrew-Kainos) [reply applications/tests/tests_existing_parties.py]: Done

File: applications/views/existing_parties.py (hunk 2 above)
Comment (BenAndrew-Kainos) [reply applications/views/existing_parties.py]: Done

File: applications/urls.py (hunk 1 above)
Comment (BenAndrew-Kainos) [reply applications/urls.py]: Done

File: applications/tests/tests_existing_parties.py (hunk 4 above)
Comment (BenAndrew-Kainos) [reply applications/tests/tests_existing_parties.py]: Done

**Pull request merged**